sentry/src/main/java/io/sentry/SentryStackTraceFactory.java

```diff
@@ -36,10 +36,17 @@ public SentryStackTraceFactory(
       sentryStackFrames = new ArrayList<>();
       for (StackTraceElement item : elements) {
         if (item != null) {
+
+          // we don't want to add our own frames
+          final String className = item.getClassName();
+          if (className.startsWith("io.sentry.")) {
+            continue;
+          }
```

Comment:
I'm not a big fan of it, but usually, people don't want to, right.
we'd need to rename our sample packages, otherwise, it'll skip all the sentry frames (from samples) too
cc @bruno-garcia

Reply:
Yeah in C# I had to rename the samples

Reply:
And document it because there folks named their tests Sentry.Bla
But I don't think it's the case in java since it's reverse domain. Still worth adding to the docs